doc/Rules.md

```diff
@@ -161,8 +161,8 @@ document:
 ```
 
 The configured list style can be a specific symbol to use (asterisk, plus, dash),
-can require that usage be consistent within the document, or can require that each
-sublist have a consistent symbol that is different from its parent list.
+can require that usage is consistent within the document, or can require that each
+sublist has a consistent symbol that is different from its parent list.
```

Comment:
I insist on the alternative I suggested [above](https://github.com/DavidAnson/markdownlint/pull/360/files#r549501823): "require each sublist to have a consistent symbol".

Reply:
Is the newer one better?:

> to ensure that all list styling is consistent, or to ensure that each
sublist has a consistent symbol that differs from its parent list.